### PR TITLE
Update section 6.1 wrt "authenticity"

### PR DESCRIPTION
**Pull request by talltree**
Revised the sentence describing "authenticity" and moved the explanation of "message integrity" to a footnote.

### CODE REVIEW

**Review by SmithSamuelM — APPROVED** [review]

File: spec.md [hunk 1]
```diff
@@ -115,14 +115,14 @@ These twin objectives led the ToIP Foundation to begin the work with the Design
...
+3. **Privacy:** will the expectations of each party with respect to usage of shared information be honored by the other parties?
 
-Note that, in some trust relationships, confidentiality and privacy may be optional. Thus our design goal with the ToIP stack is to achieve these four properties in the order listed.[^1]
+Note that, in some trust relationships, confidentiality and privacy may be optional. Thus our design goal with the ToIP stack is to achieve these three properties in the order listed.[^2]
 
-[^1]: Another standard information trust criteria is **availability**. This is a concern with the design and implementation of operational deployments of the ToIP stack and should be addressed in the associated operational governance frameworks.
+[^1]: With respect to this design goal, authenticity includes **message integrity**, i.e., a communication is not authentic if it has been tampered with in any way.
+[^2]: Another standard information trust criteria is **availability**. This is a concern with the design and implementation of operational deployments of the ToIP stack and should be addressed in the associated operational governance frameworks.
```
Comment (talltree) [inline spec.md:125]: "Another important property of the architecture is..."

Comment (talltree) [reply spec.md]: This wording change (to the start of the second footnote) was made in a second commit.

**Comment by SmithSamuelM** [comment]
Footnote says first four design principles. Should to changed to first three design principles because integrity has been subsumed by authenticity.

File: spec.md [hunk 2]
```diff
@@ -115,14 +115,14 @@ These twin objectives led the ToIP Foundation to begin the work with the Design
 
 With regard to the first design goal, establishing trust between parties requires that each party develop confidence in the following properties of their relationship:
 
-1. **Authenticity:** is the receiver of a communication able to verify that it originated from the expected sender?
-2. **Integrity:** is the receiver able to verify that a communication is complete and has not been tampered with?
-3. **Confidentiality:** is the contents of a communication protected so only authorized parties have access?
-4. **Privacy:** will the expectations of each party with respect to usage of shared information be honored by the other parties?
+1. **Authenticity:** is the receiver of a communication able to verify that it originated from the expected sender and has not been tampered with?[^1]
+2. **Confidentiality:** is the contents of a communication protected so only authorized parties have access?
```
Comment (talltree) [inline spec.md:119]: Remove "expected".

**Comment by andorsk** [comment]
Merge decided to via 11/10 meeting.

**Pull request merged**